services/awardSvc.js

```diff
@@ -4,9 +4,56 @@ function addMsgCounter(){
     return counter;
 }
 
+//var MongoClient = require('').MongoClient;
+//var url = "";
 
+const mongoose = require('mongoose');
```

Comment:
This is a business logic (The controller C in MVC) layer. Adding mongoose and the daily user post counter schema is database (model M in MVC) layer. You should just import the existing daily user post counter (look how I do it in index.ts)

So remove everything from line 10-22